### DISCUSSION TITLE
More explanation in docs about QTree "tick-strategy" accepted values.

### DISCUSSION THREAD

**Discussion by jonathanpmartins** [quasarframework/quasar#10168] [General - Components / Directives / etc] [answered]
Go to the [Qtree Docs](https://quasar.dev/vue-components/tree#qtree-api). Filter by "tick-strategy" and you will see the following explanation:

---

`tick-strategy`: String
###### Description
The type of strategy to use for the selection of the nodes
###### Default value
"none"
###### Accepted values
`none` `strict` `leaf` `leaf-filtered`

---

Would be nice if the docs had and explanation about what every accepted value does.
Keep doing the good work! The results are awesome!

**Comment by pdanpdan** [comment, marked as answer]
You mean that?
https://quasar.dev/vue-components/tree#tick-strategy

**Reply by jonathanpmartins** [reply]
uff... lost myself in the jungle! kkkkk
Thank you very much!
